lib/pymedphys/_experimental/autosegmentation/unet.py

```diff
@@ -0,0 +1,274 @@
+from pymedphys._imports import numpy as np
+from pymedphys._imports.slow import tensorflow as tf
+
+
+def unet(
+    grid_size,
+    output_channels,
+    max_filter_num=None,
+    number_of_filters_start=32,
+    min_grid_size=8,
+    num_of_fc=2,
+    batch_normalisation=True,
+    use_dropout=True,
+):
+    inputs = tf.keras.layers.Input((grid_size, grid_size, 1))
```

Comment:
docstring

Reply:
Here I might instead opt for just deleting the code altogether instead of giving it any more love. In the end I didn't use it.

...generally I prefer to leave experimental code hanging around for a bit to refer back to before deleting it though...